README.md

```diff
@@ -14,7 +14,6 @@ Please see [CONTRIBUTING.md](./CONTRIBUTING.md).
 ## Installation
 
 ```bash
-apt-get install python-dev
```

Comment:
This is already removed by https://github.com/jaegertracing/jaeger-client-python/issues/139
Not related to this. ( committed  by mistake )